pkg/types/v1/factors/types.go

```diff
@@ -37,7 +37,11 @@ type Embodied struct {
 	AdditionalCPUsKiloWattCO2e    float64 `yaml:"additionalcpus"`
 	AdditionalGPUsKiloWattCO2e    float64 `yaml:"additionalgpus"`
 	TotalEmbodiedKiloWattCO2e     float64 `yaml:"total"`
-	Architecture                  string
+	// vCPU count for instance Type
+	VCPU float64 `yaml:"vCPU"`
+	// Highest amount of vCPUs in machine type family
```

Comment:
This is a YMMV thing and you don't need to change it if you don't want to, but sometimes it can be nice to put the comment after the line so 
```
VCPU float64 `yaml:"vCPU"` // vCPU count for instance Type
```
so that the struct formatting/spacing stays the same. But since the comments are longer this may be a better approach. Just a blerp